pkg/test/utils/kubernetes.go

```diff
@@ -1071,6 +1071,32 @@ func RunCommand(ctx context.Context, namespace string, cmd harness.Command, cwd
 	return nil, err
 }
 
+// toAssertCommands converts a set of TestAssertCommand to Commands so it all the existing functions can be used 
+// that expect Commands data type.
+func toAssertCommands(assertCommands []harness.TestAssertCommand, timeout int) (commands []harness.Command) {
```

Comment:
I had not thought of the solution this way... but makes sense.
what doesn't make sense is the method name `toAssertCommands` it isn't `to` assert it is `from` assert commands and to standard commands.    Naming is hard but this one is misleading.   perhaps `convertAssertCommand`

Reply:
Renamed